pkgs/development/python-modules/fakeredis/default.nix

```diff
@@ -42,6 +45,15 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  patches = [
+    # Support for redis >= 4.1.0, https://github.com/jamesls/fakeredis/pull/324
+    (fetchpatch {
+      name = "support-redis-4.1.0.patch";
+      url = "https://github.com/jamesls/fakeredis/commit/8ef8dc6dacc9baf571d66a25ffbf0fadd7c70f78.patch";
```

Comment:
I believe that this patch actually adds support for `redis<=4.1.0` not `redis>=4.1.0`. Would you mind updating the commit message?

Reply:
Done